test/TableGateway/Feature/FeatureSetTest.php

```diff
@@ -77,4 +78,93 @@ public function testAddFeatureThatFeatureHasTableGatewayButFeatureSetDoesnotHas(
         $featureSet = new FeatureSet;
         $this->assertInstanceOf('Zend\Db\TableGateway\Feature\FeatureSet', $featureSet->addFeature($feature));
     }
+
+    /**
+     * @cover FeatureSet::canCallMagicCall
```

Comment:
`@covers` instead of `@cover` ,  also, use fully namespaced class, `@covers Zend\Db\TableGateway\Feature\FeatureSet::canCallMagicCall`